libcaf_net/test/string_application.cpp

```diff
@@ -194,12 +197,14 @@ CAF_TEST(receive) {
                   sec::unavailable_or_would_block);
   CAF_MESSAGE("adding both endpoint managers");
   auto mgr1 = make_endpoint_manager(mpx, sys, policy::scribe{sockets.first},
-                                    stream_string_application{sys, buf});
+                                    extend<string_application>::with<
```

Comment:
This rather unwieldy type is used twice. I'd give it a type alias to make the code more readable and less repetitive:

```c++
  using application_type = extend<string_application>::with<stream_string_application>;
```

Reply:
Thanks for the tip @Neverlord.